src/main/resources/data/minecraft/tags/blocks/logs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:cactus"
```

Comment:
we definitely should not do that, probably won't break other mods but may cause weird things since cactus is usually not recognised as wooden log

Reply:
true, we might only want the plank tag